visidata/data.py

```diff
@@ -9,9 +9,11 @@
 option('filetype', '', 'specify file type', replay=True)
 option('incr_base', 1.0, 'start value for column increments', replay=True)
 
+__all__ = ['SettableColumn', 'inputFilename', 'numrange', 'updateColNames', 'num', 'fillNullValues', 'inputPath', 'loadInternalSheet', 'open_txt']
+
 
 def _default_colnames():
-    'A B C .. Z AA AB .. ZZ AAA .. to infinity'
+    'a b c .. z aa ab .. zz aaa .. to infinity'
```

Comment:
why was this changed? The names are actually uppercase.

Reply:
I have no idea, I noticed that in the diff, and assumed it was you. I should have asked, I will revert it!